ql/src/java/org/apache/hadoop/hive/ql/optimizer/physical/Vectorizer.java

```diff
@@ -244,7 +247,7 @@
 
 public class Vectorizer implements PhysicalPlanResolver {
 
-  protected static transient final Logger LOG = LoggerFactory.getLogger(Vectorizer.class);
+  protected static final Logger LOG = LoggerFactory.getLogger(Vectorizer.class);
```

Comment:
why was the transient modifier removed?

Reply:
Sonar marked it as [code smell](https://sonarcloud.io/project/issues?issues=AYJpdDZw9k-CsauE5Bp5&open=AYJpdDZw9k-CsauE5Bp5&id=apache_hive) because for non-`Serializable` classes the modifier is a no-op and it's suggested to remove it.